options/locale/locale_en-US.ini

```diff
@@ -1922,6 +1922,16 @@ settings.webhook.payload = Content
 settings.webhook.body = Body
 settings.webhook.replay.description = Replay this webhook.
 settings.webhook.delivery.success = An event has been added to the delivery queue. It may take few seconds before it shows up in the delivery history.
+settings.webhook.auth_header.section = Authorization Header
+settings.webhook.auth_header.description = Add authorization header to webhook delivery.
+settings.webhook.auth_header.name = Header name
+settings.webhook.auth_header.type = Header content type
+settings.webhook.auth_header.type_basic = Basic authentication
+settings.webhook.auth_header.type_token = Token authentication
+settings.webhook.auth_header.username = Username
+settings.webhook.auth_header.password = Password
+settings.webhook.auth_header.token = Token
+settings.webhook.auth_header.token_description = During webhook delivery, the given value will be prepended with <code>token</code> followed by a space.
```

Comment:
TBH, I am not sure about the naming here. Based on MDN docs[^1][^2] the naming would be "Authentication schema" instead of "Header content type" and the "Token authentication" would better fit with "Bearer scheme".

[^1]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization
[^2]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#authentication_schemes